src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -10969,22 +10972,6 @@ and ApplyAbstractSlotInference (cenv: cenv) (envinner: TcEnv) (baseValOpt: Val o
                      | _ -> [] // check that method to override is sealed is located at CheckOverridesAreAllUsedOnce (typrelns.fs)
                       // We hit this case when it is ambiguous which abstract method is being implemented.
 
-             if g.langVersion.SupportsFeature(LanguageFeature.ErrorForNonVirtualMembersOverrides) then
```

Comment:
Was there no existing test for this?

Reply:
I think there were, but we weren't running those on preview.
Let's see what I can do with this.

Reply:
Really? I remember running then locally and making sure these  were Ok .

Reply:
No, what I meant is that we haven't had any tests running on preview version, which could've uncovered the bug